specification/sections/06-tracking.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Tracking
+description: Specification defining tracking API
+toc_max_heading_level: 4
+---
+
+# 6. Tracking
+
+[![experimental](https://img.shields.io/static/v1?label=Status&message=experimental&color=orange)](https://github.com/open-feature/spec/tree/main/specification#experimental)
+
+## Overview
+
+Experimentation is a primary use case for feature flags.
+In practice, this often means flag variants are assigned to users at random or in accordance with a business rule, while the impact of the assigned variant on some business objective is measured.
+Vendors and custom solutions often support a _tracking_ or _goal measuring_ API to facilitate the measurement of these business objectives.
+
+### Goals
+
+- Develop official terminology to support consistent implementation
+- Specify a flexible API widely compatible with basic vendor functionality
+  - Define tracking event payload
+  - Define tracking event identifier
+  - Support client and server paradigms
+
+### Non-goals
+
+- Creating an experimentation platform
+- Covering every user-tracking use case
+  - Focus on A/B testing, and experimentation based on feature flags
```

Comment:
I got confused whether focus on A/B testing is a non-goal

Reply:
I will clarify this.

Reply:
I've clarified this by adding a new goal: `Support A/B testing and experimentation use-cases`

and removing this from non-goals. Non goals are all now negative statements.